dragonmapper/html.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+"""Formatting Chinese into HTML with dragonmapper's functions"""
+
+
+from __future__ import unicode_literals
+from dragonmapper import hanzi
+from dragonmapper import transcriptions as trans
+from zhon import pinyin
+from zhon import zhuyin
+
+
+"""See recomended CSS style: DRAGONMAPPER_DIR/style.css"""
+
+
+_indentation = 0
+_line_html = ''
+_puctuation = ['，', '。', '“', '”', '：', '；', '？']
+_tones_marks = ['¯', 'ˊ', 'ˇ', 'ˋ', '˙', '1', '2', '3', '4', '5']
+
+
+def _identify(s):
+
+    """
+    Returns string of text type for HTML/CSS.
+
+    *s* is the string to identify.
+    """
+
+    if hanzi.has_chinese(s):
+        return "hanzi"
+    elif s in _puctuation:
+        return "punct"
+    elif s in _tones_marks:
+        return "tone-mark"
+    else:
+        c = trans.identify(s)
+        if c == trans.ZHUYIN:
+            return "zhuyin"
+        elif c == trans.PINYIN:
+            return "pinyin"
+        elif c == trans.UNKNOWN:
+            return "unknown"
+
+
+def _stackify(s):
+
+    """
+    Stack string for HTML formatting on the left and right of characters.
+
+    *s* is the string to "stackify".
+    """
+
+    temp_s = ""
+    for c in s:
```

Comment:
Instead of using the temp variable and the `for` loop, you could try something like:
 ```python
return '<br />'.join(list(s))
```